### ISSUE TITLE
approve plugin cannot find owners

### ISSUE THREAD

**Issue by fejta** [istio-ecosystem/authservice#40]
```yaml
{
 insertId: "zsre4bfas8ajr"  
 jsonPayload: {
  author: "cfryanr"   
  component: "hook"   
  event-GUID: "0fef05e0-eecd-11e9-87f6-8b407be6260a"   
  event-type: "pull_request"   
  file: "prow/plugins/approve/approvers/owners.go:164"   
  func: "k8s.io/test-infra/prow/plugins/approve/approvers.Owners.GetSuggestedApprovers"   
  level: "warning"   
  msg: "Couldn't find/suggest approvers for each files. Unapproved: [""]"   
  org: "istio-ecosystem"   
  plugin: "approve"   
  pr: 30   
  repo: "authservice"   
  url: "https://github.com/istio-ecosystem/authservice/pull/30"   
 }
 labels: {…}  
 logName: "projects/istio-testing/logs/hook"  
 receiveTimestamp: "2019-10-14T21:53:37.741292100Z"  
 resource: {…}  
 severity: "ERROR"  
 timestamp: "2019-10-14T21:53:33Z"  
}
```

1) You could switch to CODEOWNERS, delete OWNERS and disable the approve and lgtm plugins
2) Maybe the problem is that https://github.com/istio-ecosystem/authservice/blob/db27de68732f29db33f8e2cb2fc049a208334dc1/OWNERS#L1-L5 is not a valid yaml file? I believe there needs to be whitespace after the `-`

**Comment by cfryanr** [comment]
Thanks for creating an issue, @fejta. We're new in the Istio community and we're not sure what you're referring to here.  We see that OWNERS file that you mentioned but we don't know what that file does or what format it should be written in.  I think the OWNERS file was created by one of the admins of this repo, not by the contributors of the code in this repo. Is there someplace we (the contributors) can go to learn more?

**Comment by fejta** [comment]
https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md

**Comment by cfryanr** [comment]
Hi @cjwagner, was this issue fixed by your #52?

**Comment by cjwagner** [comment]
Indeed
/close
